presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java

```diff
@@ -142,7 +142,7 @@ public void shouldSetTimezone()
             ((PrestoConnection) connection()).setTimeZoneId(timeZoneId);
             assertConnectionTimezone(connection(), timeZoneId);
         }
-        else if (usingTeradataJdbcDriver(connection())) {
```

Comment:
I would undo here, and improve the error message, because:
 - we could have new driver in the future (god save us)
 - `usingTeradataJdbcDriver` or `usingPrestoJdbcDriver` could have a bug

Reply:
in other tests we already rely on `usingTeradataJdbcDriver` being opposite of `usingPrestoJdbcDriver` and the two covering all possibilities.
if we introduce a third driver, we will need to update more tests anyway